iotdb/src/main/java/org/apache/iotdb/db/engine/modification/Deletion.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright © 2019 Apache IoTDB(incubating) (dev@iotdb.apache.org)
```

Comment:
Maybe the "copyright 2019..." and the label "\<p\>" should be removed.
You can refer to 
https://github.com/apache/incubator-iotdb/blob/master/tsfile/src/main/java/org/apache/iotdb/tsfile/file/header/ChunkHeader.java